_layouts/documentation.html

```diff
@@ -43,7 +43,13 @@
                     <a href="/documentation/commandline">Command-line</a></li>
                     <li
                     {% if page.menu == 'api' %} class="active" {% endif %}>
-                    <a href="/documentation/api">API (Java / Android)</a></li>
+                    <a href="/documentation/api">API</a></li>
+                    <li
+                    {% if page.menu == 'api' %} class="active" {% endif %}>
+                    <a href="/documentation/api#android">API (Android)</a></li>
+                    <li
+                    {% if page.menu == 'api' %} class="active" {% endif %}>
+                    <a href="/documentation/api#spring">API (Spring)</a></li>
                     <li
```

Comment:
This changes the 1-1 mapping between navigation and pages and breaks the consistency of the convention used for the rest of the site.